scripts/Synapse/config/ecr_datastore_config.json

```diff
@@ -0,0 +1,261 @@
+        "data_type": "string",
+        "nullable": true
+    },
+    "active_problems": {
+        "fhir_path": "Bundle.entry.resource.where(resourceType='Condition').where(category.coding.code='problem-item-list')",
+        "data_type": "array",
+        "nullable": true,
+        "secondary_schema": {
+            "problem": {
+                "fhir_path": "Condition.code.coding.display",
+                "data_type": "string",
+                "nullable": true
+            },
+            "problem_date": {
+                "fhir_path": "Condition.onsetDateTime",
+                "data_type": "datetime",
+                "nullable": true
+            }
+        }
+    },
+    "labs": {
+        "fhir_path": "Bundle.entry.resource.where(resourceType='Observation').where(category.coding.code='laboratory')",
+        "data_type": "array",
+        "nullable": true,
+        "secondary_schema": {
+            "test_type": {
+                "fhir_path": "Observation.code.coding.display",
+                "data_type": "string",
+                "nullable": true
+            },
+            "test_type_code": {
+                "fhir_path": "Observation.code.coding.code",
+                "data_type": "string",
+                "nullable": true
+            },
+            "test_type_system": {
+                "fhir_path": "Observation.code.coding.system",
+                "data_type": "string",
+                "nullable": true
+            },
+            "test_result_qualitative": {
+                "fhir_path": "Observation.valueString",
+                "data_type": "string",
+                "nullable": true
+            },
+            "test_result_quantitative": {
+                "fhir_path": "Observation.valueQuantity.value",
+                "data_type": "number",
+                "nullable": true
+            },
+            "test_result_units": {
+                "fhir_path": "Observation.valueQuantity.unit",
+                "data_type": "string",
+                "nullable": true
+            },
+            "test_result_code": {
+                "fhir_path": "Observation.valueCodeableConcept.coding.code",
+                "data_type": "string",
+                "nullable": true
+            },
+            "test_result_code_display": {
+                "fhir_path": "Observation.valueCodeableConcept.coding.display",
+                "data_type": "string",
+                "nullable": true
+            },
+            "test_result_system": {
+                "fhir_path": "Observation.valueCodeableConcept.coding.system",
+                "data_type": "number",
+                "nullable": true
+            },
+            "test_result_interp": {
+                "fhir_path": "Observation.interpretation.coding.display",
+                "data_type": "string",
+                "nullable": true
+            },
```

Comment:
There are some more columns I added this morning.  Also, there will be a bunch more columns added in other sections for other stories.  I'll make sure to update this file in this repo once those schema changes land in the SDK side.

Reply:
Latest PR has even more columns added:  https://github.com/CDCgov/phdi/pull/776

Reply:
Thanks @BradySkylight! I will review tomorrow and update accordingly.

Reply:
Let's not worry about getting this perfect before merging here. We want to get something into Azure so that we can move onto updating Synapse to work with this format. We can always update the schema to its final form in a subsequent PR.